### PR TITLE
l10n: Fixed grammar

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex.

Signed-off-by: rakekniven <2069590+rakekniven@users.noreply.github.com>

### CODE REVIEW

**Review by Valdnet — APPROVED** [review]

**Comment by Valdnet** [comment]
@rakekniven What about the singular for "minute"?

**Review by tflidd — APPROVED** [review]

**Comment by rakekniven** [comment]
> In Transifex, the sentence for the singular and the plural is the same.

Yes, but the translator can translate it.

**Comment by Valdnet** [comment]
> Yes, but the translator can translate it.

A translator who knows what it is all about. Inexperienced translators translate one-to-one without changing a word.

File: src/gui/sharedialog.cpp [hunk 1]
```diff
@@ -102,7 +102,7 @@ ShareDialog::ShareDialog(QPointer<AccountState> accountState,
         const auto lockExpirationTime = _filelockState._lockTime + _filelockState._lockTimeout;
         const auto remainingTime = QDateTime::currentDateTime().secsTo(QDateTime::fromSecsSinceEpoch(lockExpirationTime));
         const auto remainingTimeInMinute = static_cast<int>(remainingTime > 0 ? remainingTime / SECONDS_PER_MINUTE : 0);
-        _ui->label_lockinfo->setText(tr("Locked by %1 - Expire in %2 minutes", "remaining time before lock expire", remainingTimeInMinute).arg(_filelockState._lockOwnerDisplayName).arg(remainingTimeInMinute));
+        _ui->label_lockinfo->setText(tr("Locked by %1 - Expires in %2 minutes", "remaining time before lock expire", remainingTimeInMinute).arg(_filelockState._lockOwnerDisplayName).arg(remainingTimeInMinute));
```
Comment (tflidd) [inline src/gui/sharedialog.cpp:105]: shouldn't the second string be:
remaining time before lock expire**s**

Comment (rakekniven) [reply src/gui/sharedialog.cpp:105]: This string is not part of the string which is translated.
See https://doc.qt.io/qt-5/i18n-source-translation.html#disambiguation for details.

But you are right. Every spelling mistake should be fixed.

**Review by mgallien — APPROVED** [review]
thanks

**Comment by rakekniven** [comment]
> @rakekniven can you have a quick look at the conflict and see if you can solve it ? would be nice to fix those strings

Done

**Pull request merged**